src/main/java/com/example/DoughnutShop.java

```diff
@@ -0,0 +1,20 @@
+package com.example;
+
+public class DoughnutShop {
+
+    public String prepareDoughnut(String doughnutType) {
+        String preparedDoughnut;
+
+        if (doughnutType.equals("Chocolate")) {
+            preparedDoughnut = "A delicious chocolate doughnut prepared!";
+        } else if (doughnutType.equals("Cinnamon")) {
+            preparedDoughnut = "A delicious cinnamon doughnut prepared!";
+        } else if (doughnutType.equals("Sugar")) {
+            preparedDoughnut = "A delightful sugar doughnut prepared!";
+        } else {
+            throw new IllegalArgumentException("We do not produce this type of doughnut");
+        }
```

Comment:
Converting this to a `switch` could make this easier to add more donuts in future :)